pkg/features/list.go

```diff
@@ -42,4 +42,7 @@ var (
 
 	// NetworkGraphPatternFly enables the PatternFly version of NetworkGraph. (used in the front-end app only)
 	NetworkGraphPatternFly = registerFeature("Enable PatternFly version of NetworkGraph", "ROX_NETWORK_GRAPH_PATTERNFLY", false)
+
+	// FullNodeScanning RHCOS full node scanning function
+	FullNodeScanning = registerFeature("Enable RHCOS full node scanning function", "ROX_FULL_NODE_SCANNING", false)
```

Comment:
Nit: node scanning is more than RHCOS node scanning. Also, the level of scanning we're implementing is not "full node" but rather OS, kernel versions and rpm packages.
I suggest to have some more specific name here.

Reply:
I am fine renaming this. Below an explanation why I am using the `Full` word in the names of other objects in this epic.

I am using `Full` in the namer (here, the proto message) as we aim at full node scan in the near future (including language components). Moreover, we do have a proto message called `NodeScan` already (for the partial node scan that is currently available) and I want to avoid confusing these two flows.